api/lanes.py

```diff
@@ -1050,12 +1050,12 @@ def default(cls, library):
         )
 
     @classmethod
-    def order_facet_to_database_field(cls, order_facet):
-        if order_facet == cls.ORDER_LAST_UPDATE:
-            from core.model import MaterializedWorkWithGenre as work_model
-            return func.greatest(work_model.last_update_time, work_model.first_appearance)
-        else:
-            return Facets.order_facet_to_database_field(order_facet)
+    def order_by(cls):
+        """Order the search results by last update time."""
+        from core.model import MaterializedWorkWithGenre as work_model
+        updated = func.greatest(work_model.availability_time, work_model.first_appearance, work_model.last_update_time)
+        work_id = work_model.works_id
+        return [updated.desc(), work_id], [updated, work_id]
```

Comment:
mv_works_for_lanes_by_recently_updated also has collection_id. Is that not needed here or added somewhere else?

Reply:
Yes, you're right, and making this change significantly speeds up the library-wide crawlable feed. I was thinking that we wanted to only show a work once if it was available through multiple collections, and maybe we do eventually, but this is better for now.